client/lib/directly/index.js

```diff
@@ -23,13 +24,13 @@ const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
 let _initialize;
 
 /**
- * Set up global variables and configuration for the Directly library.
+ * Set up global variables and configuration for the RTM widget
  *
- * @see https://cloudup.com/cySVQ9R_O6S
+ * @see https://cloudup.com/cySVQ9R_O6S for the standard setup instructions
  */
-function configureDirectlyGlobals() {
-	// Set up the global DirectlyRTM function, required for the Directly library.
-	// This snippet is pasted from Directly's setup code.
+function configure() {
+	// Set up the global DirectlyRTM function, required for the RTM widget. This snippet
+	// is pasted from Directly's setup code.
```

Comment:
So "RTM" isn't an acronym confined to just this function — this whole library enables Directly's Real Time Messaging (RTM) widget. I've improperly called it "the Directly widget" through much of the code and docs, and while currently this _is_ the only widget of this type from Directly, in the future it might not be. So it felt right to make that distinction and explicitly change "the Directly widget" to "the RTM widget", which is why there's more acronyms now.

I did expanded the acronym in the [README](https://github.com/Automattic/wp-calypso/pull/11489/files#diff-afdd115cd76c8069268c8bdc02b98bc8R4) and the [top JSDoc block](https://github.com/Automattic/wp-calypso/pull/11489/files#diff-afdd115cd76c8069268c8bdc02b98bc8R4), so I hope that would be sufficient to keep other devs unconfused, but if I'm mistaken please let me know.